src/wallet/rpcwallet.cpp

```diff
@@ -2851,7 +2851,7 @@ UniValue bumpfee(const JSONRPCRequest& request)
         if (options.exists("confTarget") && options.exists("totalFee")) {
             throw JSONRPCError(RPC_INVALID_PARAMETER, "confTarget and totalFee options should not both be set. Please provide either a confirmation target for fee estimation or an explicit total fee for the transaction.");
         } else if (options.exists("confTarget")) {
-            specifiedConfirmTarget = true;
+            ignoreGlobalPayTxFee = true;
```

Comment:
Can you add a comment here like: "If the user has explicitly set a confTarget in this rpc call then override the default logic that uses the global payTxFee instead of tx confirmation target if a payTxFee is set"